### ISSUE TITLE
Deleted items

### ISSUE THREAD

**Issue by Iplay2L8** [davqvist/Q-Tech#26]
items in basic backpack are gone after recovering tombstone. single player version .04b.  dont know if it is all tiers of backpacks. was able to recreate bug. backpack was set to auto collect. once it was picked up after breaking tombstone other items were picked up by the backpack but original items not in it.  The Death item list did not have any of the original items in the backpack listed.

**Comment by davqvist** [comment]
Yeah this was reported to the dev a while back and is not fixed yet, it was mentioned to me last week and I got no answer yet unfortunately. https://github.com/Flanks255/simplybackpacks/issues/20

**Comment by angrybirdaaron05** [comment]
this just happened to me what caused me to loose afair bit of gems
